packages/playwright-test/src/reporters/html.ts

```diff
@@ -128,14 +128,14 @@ class HtmlReporter {
     const reportFolder = htmlReportFolder(this._outputFolder);
     await removeFolders([reportFolder]);
     const builder = new HtmlBuilder(reportFolder, this.config.rootDir);
-    const ok = builder.build(reports);
+    const { ok, singleTestId } = builder.build(reports);
```

Comment:
revert the changes in this file please.

Reply:
I messed up a rebase, sorry. This change wasn't intentional :)

Reply:
I messed up a rebase, sorry. This change wasn't intentional :)